src/config.c

```diff
@@ -312,7 +316,7 @@ bool config_to_color(const char* text, argb_t* color)
     if (strlen(text) > 6) { // value with alpha (RRGGBBAA)
         *color = (num >> 8) | ARGB_SET_A(num);
     } else {
-        *color = num | ARGB_SET_A(0xff);
+        *color = num | ARGB_SET_A(0xffu);
```

Comment:
It makes the code less readable. Please, remove these `u`s.

Reply:
You are right. it was bad decision.
I have reverted this part and replaced 'u' with cast to argb_t where is needed.
Please review changed commit.